Makefile

```diff
@@ -2,7 +2,7 @@ src_path = src
 
 default: restore format build unit-tests
 
-verify: build
+verify: unit-tests # Verify code is ready for commit to branch
```

Comment:
Great work - I applaud the effort and unit-tests prior to deployment is just the icing on the cake